### PR TITLE
feat(tile-converter): Option not to use Geod transformation

### PR DESCRIPTION
**Pull request by dariaterekhova-actionengine**
Added an option not to use Geod transformation via the EGM file when converting 3d tiles to i3s

### CODE REVIEW

File: modules/tile-converter/src/converter-cli.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ type TileConversionOptions = {
    * default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib
    * https://geographiclib.sourceforge.io/html/geoid.html */
   egm: string;
+  /** 3DTiles->I3S only. Whether the converter uses Earth Gravity Model (*.pgm) */
+  noEgm: boolean;
```
Comment (ibgreen) [inline modules/tile-converter/src/converter-cli.ts:40]: Maybe just allow egm to be omitted and that means no egm is applied?
```suggestion
   * https://geographiclib.sourceforge.io/html/geoid.html.
   * If not supplied, no EGM correction is applied. */
  egm?: string;
```

**Review by ibgreen — APPROVED** [review]

File: modules/tile-converter/src/converter-cli.ts (hunk 1 above)
Comment (dariaterekhova-actionengine) [reply modules/tile-converter/src/converter-cli.ts:40]: We're applying default egm in this case, so we needed something different for no-egm

Comment (ibgreen) [reply modules/tile-converter/src/converter-cli.ts:40]: egm: 'none' ?

Comment (dariaterekhova-actionengine) [reply modules/tile-converter/src/converter-cli.ts:40]: Looks much better but I already pushed the PR, so I'll change it later

**Pull request merged**